postman/Create and Schedule Boadcast.postman_collection.json

```diff
@@ -0,0 +1,542 @@
+{
+	"info": {
+		"_postman_id": "91242393-418c-49a2-947e-b46584d95a0a",
+		"name": "Create and Schedule Boadcast",
+		"description": "This will get your account ID, select a random list and create a scheduled broadcast",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [{
+			"name": "Get Account",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "74e7dce1-167a-4675-8431-bdf2223fcd25",
+					"exec": [
+						"pm.test(\"Check consumer_key is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"consumer_key\")).to.equal(true) && pm.expect(pm.environment.get(\"consumer_key\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check consumer_secret is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"consumer_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"consumer_secret\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check access_token is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"access_token\")).to.eql(true) && pm.expect(pm.environment.get(\"access_token\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check token_secret is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"token_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"token_secret\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Successfully get an account Id\", function(){",
+						"    pm.response.to.have.status(200);",
+						"    var jsonData = pm.response.json();",
+						"    pm.expect(jsonData.entries.length).to.be.above(0);",
+						"    pm.globals.set(\"accountId\",jsonData.entries[0].id ) ;   ",
+						"});",
+						"",
+						"pm.test(\"Successfully submitted request\", function(){",
+						"    pm.response.to.have.status(200);",
+						"});"
+					],
+					"type": "text/javascript"
+				}
+			}],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "f6163948-14a0-48e5-ad18-ee37ab1a3c0d",
+					"exec": [
+						"pm.test(\"Successfully submitted request\", function(){",
+						"    pm.response.to.have.status(200);",
+						"});",
+						"",
+						"pm.test(\"At least one list found\", function() {",
+						"    var jsonData = pm.response.json();",
+						"    pm.expect(jsonData.total_size).to.be.above(0);",
+						"    var listId = Math.floor((Math.random() * jsonData.total_size ));",
```

Comment:
1) This variable isn't an id, it's an index into the entire list collection (which isn't good because...)
2) Only the first 100 lists are included in this page, while total_size shows the total number of lists on the account. This should instead pick a random number between 0 and the size of the entries array.

Reply:
Updated test to check entry array length